tools/parse-glibc.py

```diff
@@ -19,9 +19,9 @@ def checknonnull(cfg, functionName, nonnull):
             continue
         argpos2 = functionCfg.find('</arg>', argpos1)
         notnullpos = functionCfg.find('not-null', argpos1)
-        if notnullpos > 0 and notnullpos < argpos2:
+        if 0 <= notnullpos < argpos2:

```

Comment:
the main change is that >0 is change to >= 0.
If you don't like a<b<c, I can write it like a<b and b<c, but I think that the first one is really pythonic and kind of easy-to-write and easy-to-read.